Intersect.Client/Entities/Entity.cs

```diff
@@ -1386,6 +1386,147 @@ public virtual void DrawName(Color textColor, Color borderColor = null, Color ba
             );
         }
 
+        // Tags
+        public virtual void DrawTag(NpcTagPos npcTagPos, PlayerTagPos playerTagPos)
+        {
+            // Variables
+            string tagFileName = "";
+            string entityName = this.Name;
+            bool customNpcTag = Options.Npc.CustomTagIcons.Contains(entityName);
+            bool customPlayerTag = Options.Player.CustomTagIcons.Contains(entityName);
+            bool customNpcTagOnly = Options.Npc.ShowCustomTagsOnly;
+            var nameSize = Graphics.Renderer.MeasureText(entityName, Graphics.EntityNameFont, 1);
+            var nameCentHorPos = (int)Math.Ceiling(GetCenterPos().X);
+            var nameVertPos = GetLabelLocation(LabelType.Name);
+            float x;
+            float y;
+            // Feature check
+            if (!Options.Npc.ShowTags || !Options.Player.ShowTags)
+            {
+                return;
+            }
+            // Npc Tags
+            if (Options.Npc.ShowTags && !(this is Player))
+            {
+                // Custom Npc Tag
+                if (customNpcTag)
+                {
+                    tagFileName = $@"Npc_{entityName}.png";
+                }
+                // Default Npc Tags.
+                if (!customNpcTag && !customNpcTagOnly)
```

Comment:
Make this `else if (!customNpcTagOnly)`, the check for `customNpcTag` was already done in the previous if statement.

Reply:
👍🏽 
```
            // Feature and Entity check
            if (!Options.Npc.ShowTags || this is Player)
            {
                return;
            }
            // Custom Npc Tag
            if (customNpcTag)
            {
                tagFileName = $@"Npc_{entityName}.png";
            }
            // Default Npc Tags.
            else if (!customNpcTagOnly)
            {
```